snippets/scripts/components-forge-field-to-compass-components/migrateCompassCFToComponent.py

```diff
@@ -258,25 +266,29 @@ def main():
     compass_custom_field_fieldid = compass_formatted_custom_field.field_id
 
     # Step2: Get the list of related issues that contains the custom field values
+    # project_key is an optional field here.
+    # By default, we set empty which means you can migrate the issues cross all projects.
+    # If you want to migrate custom fields on single project,
+    # you can put PROJECT_KEY instead of empty string in last variable.
     issues = get_related_issues(
-        DOMAIN_NAME, USER_NAME, API_TOKEN, compass_custom_field_customid, 0, maxResults
+        DOMAIN_NAME, USER_NAME, API_TOKEN, compass_custom_field_customid, 0, ''
```

Comment:
? you're always passing empty string for project key instead of the variable.

Reply:
>     # If you want to migrate custom fields on single project,
>    # you can put PROJECT_KEY instead of empty string in last variable.

Wait you're still expecting the customer to change the code? Is
> could you make the project input optional (i.e. Enter your project key (or leave empty for all projects): ) then in the code, you can check whether project_key is empty, if so, do not include it in the search query.

not possible to do?